src/types/notifications.ts

```diff
@@ -17,7 +17,9 @@ interface Deployment {
   phase: DeploymentPhase;
   state: DeploymentState;
   sourceBranchName: string;
+  sourceCommit: string;
   targetBranchName: string;
+  mergeBranchName: string;
```

Comment:
I suggest to call this `deploymentBranchName` on the public API, for clarity.

Reply:
Good idea, updated in c07067fad6f8727637f5fcf88c16d9d023bfced1